akd/src/ecvrf/ecvrf_impl.rs

```diff
@@ -54,8 +54,17 @@ impl core::ops::Deref for VRFPrivateKey {
         &self.0
     }
 }
+
+impl std::clone::Clone for VRFPrivateKey {
+    fn clone(&self) -> Self {
+        // In theory, creating a key from bytes could be a DecodingError, except
+        // we just copied these bytes out of the source key, so ...
+        Self(ed25519_PrivateKey::from_bytes(self.0.as_bytes()).unwrap())
```

Comment:
This feels a little icky, using `unwrap()` here. New to Rust, I couldn't find good guidance about how to handle an error occurring in `clone()`. It seems that getting an error here borders on impossible, seeing as it creates a key from the bytes of another key, so is the panic the right approach here?

Reply:
Yep exactly, here we would probably want a panic. Since it should theoretically be impossible. Unfortunately we can't really write "can't panic" :/